### ISSUE TITLE
updating avatar of group doesn't update its entry on the groups index page

### ISSUE THREAD

**Issue by ara4n** [vector-im/riot-web#5626]

**Comment by lampholder** [comment]
Is this (just) (another) a caching issue? @lukebarnard1 ?

**Comment by lukebarnard1** [comment]
Yes, we cache group profiles for 30mins (it's the same cache we use for Flair)

**Comment by ara4n** [comment]
Somewhere I remember having a big discussion about the 'caching problem' but i'm entirely failing to find it now. https://github.com/vector-im/riot-web/issues/5870 and https://github.com/vector-im/riot-web/issues/5235 are related, but don't have the details.

**Comment by ara4n** [comment]
...and https://github.com/vector-im/riot-web/issues/5228....

**Comment by turt2live** [comment]
Looks like this particular caching issue got fixed at some point (can no longer reproduce).
